hubtools/Cargo.toml

```diff
@@ -13,3 +14,5 @@ zip = { version = "0.6", default-features = false, features = ["bzip2"] }
 
 tlvc = { git = "https://github.com/oxidecomputer/tlvc", default-features = false }
 tlvc-text = { git = "https://github.com/oxidecomputer/tlvc", default-features = false }
+lpc55_sign = { git = "https://github.com/oxidecomputer/lpc55_support", default-features = false, branch = "library-ify" }
```

Comment:
TODO: revert these to `main` once https://github.com/oxidecomputer/lpc55_support/pull/31 is merged